internal/providers/terraform/parser.go

```diff
@@ -99,7 +99,12 @@ func (p *Parser) parseJSONResources(parsePrior bool, baseResources []*schema.Res
 	p.stripDataResources(resData)
 
 	for _, d := range resData {
-		if r := p.createResource(d, usage[d.Address]); r != nil {
+		usageAddress := usage[d.Address]
+
+		if arrayUsageAddress := usage[fmt.Sprintf("%s[*]", strings.Split(d.Address, "[")[0])]; arrayUsageAddress != nil {
```

Comment:
Would this work with counts in modules? If the module itself uses `count` or `for_each` and the resource has the same then I think the address comes through as `module.my_module[0].aws_instance.my_instance[2]`